script.js

```diff
@@ -0,0 +1,53 @@
+let Books = [];
+
+const addBook = () => {
+  const title = document.getElementById('title').value;
+  const author = document.getElementById('author').value;
+  const book = { id: new Date().getTime(), title, author };
+  if (localStorage.getItem('booksData') !== null) {
+    Books = JSON.parse(localStorage.getItem('booksData'));
+    Books.push(book);
+    const convertedBooks = JSON.stringify(Books);
+    localStorage.setItem('booksData', convertedBooks);
+    window.location.reload();
+  } else {
+    Books.push(book);
+    const convertedBooks = JSON.stringify(Books);
+    localStorage.setItem('booksData', convertedBooks);
+    window.location.reload();
+  }
+};
+
+// eslint-disable-next-line no-unused-vars
+function removeBook(id) {
```

Comment:
- [ ] To follow Javascript best practices, kindly use the arrow function. :pray: